contracts/interface/IOtcMakerStruct.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Change the license to `GPL-3.0-or-later`

Reply:
Nice catch!!!